src/tree/ResolvedFunctionAppResource.ts

```diff
@@ -190,12 +190,24 @@ export class ResolvedFunctionAppResource implements ResolvedAppResourceBase {
         const sourceControl: SiteSourceControl = await client.getSourceControl();
         const proxyTree: SlotTreeItem = this as unknown as SlotTreeItem;
 
-        this.deploymentsNode = new DeploymentsTreeItem(proxyTree, this.site, siteConfig, sourceControl);
+        this.deploymentsNode = new DeploymentsTreeItem(proxyTree, {
+            site: this.site,
+            siteConfig,
+            sourceControl,
+            contextValuesToAdd: ['azureFunctions']
```

Comment:
I think it makes sense to either use `azureFunctions` everywhere in this repo or `azFunc`.  Mixing and matching could get confusing.

Reply:
Ok I switched them all to `azFunc`. I'd prefer `azureFunctions`, but there is already a lot of usages of `azFunc` and I'd rather not introduce (more) new bugs.